bin/pd-queue

```diff
@@ -101,8 +101,7 @@ def _status(agent_config, _, args):
     # key1                                        1         0         0
     # key2                                        1         0         1
 
-    from six.moves import zip_longest
-
+    from pdagent.thirdparty.six.moves import zip_longest
```

Comment:
🤔 I'm guessing this helps with consistency in packaging to a binary.

Reply:
For constructing a table when outputting status. There are possibly more graceful ways of doing this, but trying to avoid shuffling too much.

Did realize in taking another look that we're still trying `itertools` fallbacks in `pd-queue` -- removing those since this should make them unnecessary.

Reply:
Oh I meant more, putting the third parties in a `thirdparty` directory -- usually you _can_ just do something like `from six.moves`.

Reply:
Ah, misunderstanding on my part!

We were having PYTHONPATH weirdness with our CentOS tests where `six` wasn't discoverable and moving it ends up being one of the easier paths to resolve.

There are a couple other places we could put it, but we already have this `thirdparty` convention so decided to follow it.